src/css/imports/components.scss

```diff
@@ -37,7 +37,7 @@
 @import "../../components/basic_search/css/component.scss";
 @import "../../components/body/css/component.scss";
 @import "../../components/breadcrumbs/css/component.scss";
-@import "../../components/callout/css/component.scss";
+@import "../../components/callout/callout.scss";
```

Comment:
just checking, was this changed by the build script or by hand?

it should be picked up by https://github.com/qld-gov-au/qgds-vanilla/blob/main/.esbuild/plugins/qgds-plugin-sass-glob-builder.js#L9 on ``npm run build`` to include all scss from all components.